app/models/miq_worker.rb

```diff
@@ -52,6 +53,14 @@ def self.workers
     workers_configured_count
   end
 
+  def self.singleton_worker?
```

Comment:
Alternately, we could have a method called `scalable?` that is just the opposite of this?  While either would be accurate, I wonder if `scalable?` is more readable in the contexts we care about.

Reply:
hmm, naming is hard.  I like scalable but it seems a somewhat vague especially if not familiar with our workers. 
 Scalable can be size (count/memory threshold) or even "(of a computing process) able to be used or produced in a range of capabilities" such as adding/removing roles.  I know what's meant here since scaling up for us usually means number of worker processes but maybe `count_scalable?` or some other term would be more precise for what we're scaling.

Reply:
Okay went with `#scalable?`